requirements/base.txt

```diff
@@ -1,31 +1,34 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
```

Comment:
Is using an earlier python version (going backward instead of forward) intentional/important? Or does it just reflect an environment difference between your local and that of the last person who ran this?